src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -1728,6 +1728,81 @@ public void sendLeaveMessage(Player player, String quitMessage) {
         }
     }
 
+    
+    /**
+     * Gives an "online" role to a player after joining. The role id can be added in the config.
+     *
+     * @param player the player
+     * @see #removePlayerOnlineRole(Player)
+     */
+    public void givePlayerOnlineRole(Player player) {
+        if (player == null) throw new IllegalArgumentException("player cannot be null");
+
+        // retrieves the role if possible, if not returns null
+        String roleName = config().getString("OnlineRoleName");
+        Role role = DiscordUtil.getRoleByName(getMainGuild(), roleName);
```

Comment:
This needs to have a short circuit to not continue if the role name is the default `OnlineRoleName` because certainly not everyone will set it

Reply:
I can add it, but is it really necessary? Considering the fact that the function is only called from `playerJoin/LeaveListeners` if `OnlineRoleName` is set to the non-default value in the config in the first place.

Reply:
Never mind then